### DISCUSSION TITLE
Can 2 independent plots be overlayed on top of another

### DISCUSSION THREAD

**Discussion by ogencoglu** [vega/altair#3722] [General]
For example the minimap example: https://altair-viz.github.io/gallery/scatter_with_minimap.html

Can the minimap (or a completely independent chart) be on top of the larger on instead of at the right side?

**Comment by joelostblom** [comment]
I think it is only possible if they charts have the same dimensions

**Reply by ogencoglu** [reply]
I see. Thanks for the reply.
